Trimble.Modus.Components/Controls/TMCheckBox/TMCheckBox.cs

```diff
@@ -140,14 +142,14 @@ private static void OnIsCheckedChanged(BindableObject bindable, object oldValue,
             var customCheckboxView = (TMCheckBox)bindable;
             if (customCheckboxView.IsChecked)
             {
-                customCheckboxView._checkbox.Source = ImageSource.FromFile(ImageConstants.CheckedCheckBoxButton);
-
+                customCheckboxView._checkbox.SetAppTheme<FileImageSource>(
```

Comment:
Here also, try the VisualStateManager. The checkbox can have Normal/Checked/Interminate states

Reply:
As TMChips doesn't have a xaml I wasn't able to proceed with that approach. Is it possible without xaml?

Reply:
Yes. You can still have a TMChipsStyles.xaml and use it in code